src/lib/services/transfer/service/endpoint.ts

```diff
@@ -28,6 +34,74 @@ export const get = function (
   }
 >;
 
+export type CreatePayload = Operations['CreateEndpoint']['payload'];
+export type CreateResponse = Operations['CreateEndpoint']['response'];
+/**
+ * Create a Globus Connect Personal guest collection.
+ * As of 2024-04-17, this method (and the Transfer API) only supports creating Globus Connect Personal guest collections.
+ *
+ * Globus Connect Personal mapped collections are created from the Globus Connect Personal application.
+ * All GCSv5 collections should be created using Globus Connect Server (i.e. {@link "@globus/sdk".gcs.collections.create | `gcs.collections.create`}).
+ *
+ * @see https://docs.globus.org/api/transfer/gcp_management/#create_guest_collection
+ */
+export const create = function (options?, sdkOptions?): Promise<JSONFetchResponse<CreateResponse>> {
+  if (options?.payload) {
```

Comment:
I tried to override the optional `ServiceMethod` `options` argument here to make it required, but didn't find a way to do it. It seems like there must be a nice way to say that one of these functions has a required `{ payload }`.